MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -34,6 +37,12 @@ public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName, Custo
 
         if (player instanceof SimpleExoPlayer) {
             ((SimpleExoPlayer) player).addAnalyticsListener(this);
+            if (player.isPlaying()) {
+                // playback started before mux got intialized
+                play();
+                buffering();
+                playing();
+            }
```

Comment:
Is there a reason that in 2.10.x we do not need to check the other state? In the other versions, we check for two states (one that is already playing and one that is not), but not here. Just want to make sure I understand the change here.